tests/system/providers/google/cloud/bigquery/example_bigquery_queries_async.py

```diff
@@ -0,0 +1,269 @@
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""
+Example Airflow DAG for Google BigQuery service.
+Uses Async version of BigQueryInsertJobOperator and BigQueryCheckOperator.
+"""
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.operators.bash import BashOperator
+from airflow.providers.google.cloud.operators.bigquery import (
+    BigQueryCheckOperator,
+    BigQueryCreateEmptyDatasetOperator,
+    BigQueryCreateEmptyTableOperator,
+    BigQueryDeleteDatasetOperator,
+    BigQueryGetDataOperator,
+    BigQueryInsertJobOperator,
+    BigQueryIntervalCheckOperator,
+    BigQueryValueCheckOperator,
+)
+from airflow.utils.trigger_rule import TriggerRule
+
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT")
+
+DAG_ID = "bigquery_queries_async"
+
+DATASET_NAME = f"dataset_{DAG_ID}_{ENV_ID}"
+LOCATION = "us"
+
+TABLE_1 = "table1"
+TABLE_2 = "table2"
+
+SCHEMA = [
+    {"name": "value", "type": "INTEGER", "mode": "REQUIRED"},
+    {"name": "name", "type": "STRING", "mode": "NULLABLE"},
+    {"name": "ds", "type": "STRING", "mode": "NULLABLE"},
+]
+
+DATASET = DATASET_NAME
+INSERT_DATE = datetime.now().strftime("%Y-%m-%d")
+INSERT_ROWS_QUERY = (
+    f"INSERT {DATASET}.{TABLE_1} VALUES "
+    f"(42, 'monthy python', '{INSERT_DATE}'), "
+    f"(42, 'fishy fish', '{INSERT_DATE}');"
+)
+
+
+CONFIGURATION = {
+    "query": {
+        "query": f"""DECLARE success BOOL;
+        DECLARE size_bytes INT64;
+        DECLARE row_count INT64;
+        DECLARE DELAY_TIME DATETIME;
+        DECLARE WAIT STRING;
+        SET success = FALSE;
+
+        SELECT row_count = (SELECT row_count FROM {DATASET}.__TABLES__ WHERE table_id='NON_EXISTING_TABLE');
+        IF row_count > 0  THEN
+            SELECT 'Table Exists!' as message, retry_count as retries;
+            SET success = TRUE;
+        ELSE
+            SELECT 'Table does not exist' as message, row_count;
+            SET WAIT = 'TRUE';
+            SET DELAY_TIME = DATETIME_ADD(CURRENT_DATETIME,INTERVAL 1 MINUTE);
+            WHILE WAIT = 'TRUE' DO
+                IF (DELAY_TIME < CURRENT_DATETIME) THEN
+                    SET WAIT = 'FALSE';
+                END IF;
+            END WHILE;
+        END IF;""",
+        "useLegacySql": False,
+    }
+}
+
+
+default_args = {
+    "execution_timeout": timedelta(hours=6),
+    "retries": 2,
+    "retry_delay": timedelta(seconds=60),
+}
+
+with DAG(
+    dag_id=DAG_ID,
+    schedule_interval=None,
```

Comment:
It was renamed to ``schedule`` recently. And None schedule will not run the test, so unless it should be skipped it should be "@once".